app/src/main/java/com/mobven/extensions/MenuFragment.kt

```diff
@@ -22,6 +26,15 @@ class MenuFragment : Fragment() {
         return binding.root
     }
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        permissionChecker = requireActivity().requestPermissions { permissions ->
+            permissions?.entries?.forEach {
+                Log.d("Permissions","${it.key} is ${it.value}")
```

Comment:
Şunu Toast'a basalım en azından app'in içinde görünsün.

Reply:
toasti da ekledim